samples/client/petstore/typescript-angular-v11-oneOf/builds/default/api/default.service.ts

```diff
@@ -168,7 +168,7 @@ export class DefaultService {
         }
 
         return this.httpClient.put<any>(`${this.configuration.basePath}/`,
-            body,
+            UNKNOWN_PARAM_NAME,
```

Comment:
@tjquinno Is your change based on the latest master?

Can you pleas regenerate the samples by running `./bin/utils/ensure-up-to-date` ?

Reply:
@wing328 It was when I started it, and again when I rebased a week ago. But it seems in the time since my branch has fallen behind so I will rebase it and rerun.

Reply:
@wing328 Just pushed to update a generated file. Please re-approve the workflow when you can.